package.json

```diff
@@ -9,7 +9,6 @@
     "test:watch": "jest --watch",
     "build": "NODE_ENV=production webpack --config webpack.prod.js",
     "build:scripts": "NODE_ENV=production webpack --config webpack.cli.js",
-    "prepublish": "npm run build",
```

Comment:
eh... maybe? I find prepublish scripts like this super annoying. But I can add it back.